include/functions/math.hpp

```diff
@@ -182,4 +182,51 @@ BigInt gcd(const long long& num1, const BigInt& num2){
 BigInt gcd(const std::string& num1, const BigInt& num2){
     return gcd(num2, BigInt(num1));
 }
+
+/*    
+    lcm (BigInt, BigInt) :
+    --------------------
+*/
+
+BigInt lcm(const BigInt &num1, const BigInt &num2){
+    // Trivial Case handling
+    if(abs_num1 == 0 || abs_num2 == 0) return 0;
```

Comment:
This is not required.

Reply:
if both are zero then gcd is zero and it removing this gives  division by zero error.

Reply:
Ah, yes, I missed this point. But don't forget to remove the `abs_` prefix or it won't compile.